src/user/commands/get-user-info.command.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/database/prisma.service';
+import { UserInfoDTO } from '../dto/user-info.dto';
+
+@Injectable()
+export class GetUserInfo {
```

Comment:
Comando tem que ter o prefixxo `Command`:
```suggestion
export class GetUserInfoCommand {
```